adserver/models.py

```diff
@@ -914,7 +914,11 @@ def show_to_keywords(self, keywords):
         # Check topics (groupings of keywords)
         if self.included_topics:
             topics = Topic.load_from_cache()
-            topic_keyword_set = set(*[topics[slug] for slug in self.included_topics])
+            topic_keyword_set = set()
+            for topic_slug in self.included_topics:
+                for kw in topics[topic_slug]:
+                    topic_keyword_set.add(kw)
```

Comment:
This further validates my mistrust of highly functional code 😀

Reply:
Can't argue with you there. I think I could have fixed it with a compound comprehension. However, trying to be clever is how I created this mess.